spec/features/quizzes_spec.rb

```diff
@@ -33,6 +33,10 @@
       # quiz show page expectations without participant
       expect(page).not_to have_text(participant_name)
 
+      # NOTE: Sleeping for 1 second here makes the login below about 10 seconds
+      # faster, and it might also help these specs not to flake on my machine.
+      sleep(1)
```

Comment:
Below, within `using_session('Quiz participant session')`, we call `sign_in(participant)`.

Without this sleep, I think that call maybe sometimes (initially?) happens within the context of this main Capybara session. I think that maybe the big advantage of this sleep is that it makes sure that this main session is "done" before we enter into the `using_session` block and sign in the quiz participant. Therefore, without this sleep, we sometimes get the quiz owner and quiz participant sessions mixed together, which can produce some really confusing failures.